packages/cli/generators/openapi/templates/src/controllers/controller-template.ts.ejs

```diff
@@ -19,8 +23,12 @@ imports.forEach(i => {
  */
 <%- decoration %>
 export class <%- className %> {
-  constructor() {}
-
+  <% if(true){ %>
```

Comment:
Could you clarify the purpose of the conditional statement? Looks like it will always resolve to `true`.